### ISSUE TITLE
.editorconfig - indexer brackets at call-site

### ISSUE THREAD

**Issue by yaakov-h** [dotnet/roslyn#33134]
Is there a .editorconfig option to control the spacing between indexer brackets? The option I'm after would:

(a) produce a diagnostic and fix for:
```csharp
array[index]
```

(b) produce no diagnostic, and VS would generate code as:
```csharp
array[ index ]
```

though some people would want the exact opposite setting.

**Comment by CyrusNajmabadi** [comment]
csharp_space_between_square_brackets?

**Comment by yaakov-h** [comment]
Using Visual Studio Community v15.9.4, that option and several others seem to have no effect.

See: https://github.com/yaakov-h/SteamKit/tree/editorconfig

If I open SteamKit2/SteamKit2.sln from there, I expect to see all sorts of warnings and errors related to spacing, as per [the editorconfig](https://github.com/yaakov-h/SteamKit/blob/editorconfig/.editorconfig#L24-L35).

Instead, after VS finished background analysis, there are no warnings, errors, or messages show in the Error List pane.

**Comment by yaakov-h** [comment]
Created #33231 as related to can't-get-anything-to-work.

**Comment by sharwell** [comment]
This is another duplicate of #19055 (invalid .editorconfig items are silently ignored, but should show a warning). See https://github.com/dotnet/roslyn/issues/33231#issuecomment-461682920 for more information.
